SwrveComponents/SwrveNode.brs

```diff
@@ -1,17 +1,30 @@
 function init()
-    m.top.observeField("startSwrve", "onStartSwrve")
+	m.top.observeField("startSwrve", "SwrveOnStartSwrve")
 
-    cfg = {
-        appId: "<Your ID>", 
-        apikey: "<Your Key>", 
-        debug: true 'Used for debug logs'
-    }
+	cfg = {
+		appId: "<Your ID>",
+		apiKey: "<Your Key>",
+		logLevel: 3 'Used for debug logs, levels ranges are: off 0, error 1, warn 2, info 3, debug 4, verbose 5 or higher'
+	}
 
-    Swrve(cfg)
+	date = CreateObject("roDateTime")
+	m.top.addFields({ startSeconds: date.AsSeconds(), startMilli: date.GetMilliseconds(), appStartSecondsDesc: (date.AsSeconds() + (date.GetMilliseconds() / 1000)) - 1000000000 })
+	m.global.addFields({ "Swrve": m.top })
+
+	if cfg.appId.Instr("Your ID") = -1 AND cfg.apiKey.Instr("Your Key") = -1 then
```

Comment:
Developers not longer need to modify this file to change the configuration values. This is important for people using Swrve as a Submodule. Also this would even allow apps to pull a configuration from a remote location if desired and supply the Swrve SDK with the needed parameters from that.

Reply:
Interesting. Is this a requirement for you setup?

Reply:
@KevBrodigan yes and we believe the requirement to edit the SDK it self to be less than ideal. This check was added so that if someone else has a system in place where they replace the `<Your ID>` and `<Your Key>` as part of their build process it should still work for them. 

To give some context as to how I had this set up for us:

1. The Swrve SDK is added as a sub module in our repo
2. As part of our build work flow we pull the Swrve files into our staging folder into the correct location
3. Make any modifications to our configuration files based on the environment we are building, for example different `appId` and `apiKey` for qa vs prod
4. read that config in later and pass those values into Swrve

We don't need the support of pulling that info from a remote source but there are many channels that have a json hosted remotely that they pull in at boot so this is something others might use and is supported as a side effect of this change.

Reply:
Ah yeah cool 👍